### ISSUE TITLE
Django 2.0 autocomplete compatibility

### ISSUE THREAD

**Issue by matijakolaric** [sehmaschine/django-grappelli#850]
Django Admin has autocomplete built in since 2.0. With all due respect to Grappelli, there are some advantages to it compared to Grappelli's. For me, the most important being that searched fields on related model are defined for the admin class, not the model. If there is a need for separate admin sites, this is a lifesaver.

The reason I am posting this and not the pull request is that I would propose to move towards Django Admin solution as being the default and keep the existing Grappelli approach as a fallback, as well as taking any advice before I dig in.

**Comment by sehmaschine** [comment]
Good point. I'm actually in favour of removing the autocompletes with grappelli.

**Comment by matijakolaric** [comment]
Well, since there is no overlapping that I can see at the first glance, it is down to templates/JS/CSS, right?
So people can choose which ones to use?

**Comment by Zax337** [comment]
I'm having an issue when using select2, it complains that no instance of jQuery is defined.
If I manually override it with `django.jQuery` it works though. Anybody else having that issue?

**Comment by sonarun** [comment]
I've been using the Django's default autocomplete since version 2.0 came out. However, I am not getting a reinitialization of the Select2 widget when adding a new line on an inline model. There was a problem in the original Django that was fixed but I don't know if there is some conflict with Grappelli on this. I am using Django 2.1.2 and the latest Grappelli.

https://code.djangoproject.com/ticket/28871
https://github.com/django/django/commit/81057645f61fe545f4f11737dbd3040043ed2436

**Comment by sonarun** [comment]
Thank you so much Sehmaschine, this upgrade seems to fix everything.

**Comment by StevenMapes** [comment]
This bug is still there, or at least back with Django 3.0. Without Grappelli you can use inline tabular forms and can choose "add another" and the new rows pick up the autocomplete. With Grappelli installed it only works for the first row

**Comment by sehmaschine** [comment]
@StevenMapes you mean with grappelli 2.14.1 (branch stable/2.14.x)? because the current releases are not yet compatible with django 3.0.

**Comment by StevenMapes** [comment]
Yes, though I need to upgrade to the latest stable branch, I thought it may be helpful to know what parts have compatibility issues.

**Comment by sehmaschine** [comment]
@StevenMapes sorry, but I don't quite get your answer. you say "yes" – so you use grappelli 2.14.1 with stable/2.14.x, right? but what do you mean (in that case) with "though I need to upgrade to the latest stable branch"?

**Comment by StevenMapes** [comment]
I thought I was testing with the latest 2.14.x branch but it turned out the project I had open was still using a 2.13.x branch (I'm working on about five projects at the moment across various versions of django from 1.11, 2.2 and 3).  This current one was suppose to be on the latest versions of everything I was using but it was behind.

I've moved it to the latest 2.14.x branch and so have now run it against both but, as you have said, you it's not fully Django3 compatible but I noticed your bug report on that didn't list specific things you were looking at whereas this bug was directly relating to one that I found which was why I commented here and not on the other entry.
